src/aiperf/dataset/mmap_cache.py

```
@@ -704,6 +704,22 @@ def _public_dataset_source_from_run(run: BenchmarkRun) -> dict[str, object] | No
+def _graph_format_from_run(run: BenchmarkRun) -> str | None:
+    """Resolved graph adapter format for ``run``, or None for a non-graph run.
+
+    Detection failures degrade to None (same as every other
+    ``resolve_graph_workload`` consumer): a key that omits graph-ness is the
+    pre-existing behavior, never worse than it.
+    """
+    from aiperf.dataset.graph.workload_detect import resolve_graph_workload
+
+    try:
+        ref = resolve_graph_workload(run)
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)